### ISSUE TITLE
Extensions: author, the rating, Repository, and License are not reachable by screen reader

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#61616]
Environment Details:
VSCode Version : 1.25.1
OS Version : Win10,MAS2.1.1,Keyboard

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Extension" button of "ActivityBar"->Enter "PowerShell" in "Search Extension" text field and click on "Install" button.
3.Navigate through keyboard such as "Tab" key.
4.Observed that whether the focus goes to Repository and License .

**Actual:**
While navigating through the key, the keyboard focus does not go to Repository and License.

**Expected:**
The focus should go to Repository and License though these are interactive and should be accessible through keyboard.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)

**User Impact:**
User will not able to access the interactive controls.

**MAS Reference:**
[https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7Bebdc3fe1-09a9-4aa2-a29b-e97764774e6c%7D](url)
**Attachment For Reference:**

[2726805.pptx](https://github.com/Microsoft/vscode/files/2506291/2726805.pptx)

**Comment by isidorn** [comment]
Should be fixed in latest if you still see it please let me know and I will reopen

**Comment by isidorn** [comment]
The issue here is that in the extensions view the author, rating, repository and license can not be reached via tab.
Reopening and assigning to @sandy081 

![image](https://user-images.githubusercontent.com/1926584/56122351-868feb00-5f72-11e9-8b31-1fb605f844b8.png)

**Comment by isidorn** [comment]
Details, Contibution, Changelog, Dependencies should also be reachable by tab.

This feels like low hanging fruit. Can you provide some code pointers for this @sandy081 
We can add help-wanted and I could also look into it if nobody from the community jumps on it.

**Comment by sandy081** [comment]
Sure - https://github.com/Microsoft/vscode/blob/master/src/vs/workbench/contrib/extensions/electron-browser/extensionEditor.ts#L396

**Comment by isidorn** [comment]
All these elements are now reachable by tab.

Follow up work: we should polish the aria labels so the descriptions are better, and we should make it possible to trigger those elements via `space`.

**Comment by sandy081** [comment]
Thanks @isidorn
